hack/kind-up.sh

```diff
@@ -123,6 +123,7 @@ setup_containerd_registry_mirrors() {
   REGISTRY_HOSTNAME="garden.local.gardener.cloud"
 
   for NODE in $(kind get nodes --name="$CLUSTER_NAME"); do
+    echo "Setting up containerd registry mirrors on node ${NODE}.";
```

Comment:
I would rather suggest to add this info to L144 such as:
```
  echo "[${NODE}] Setting up containerd registry mirror for host ${UPSTREAM_HOST}.";
```

Reply:
Sure. That was the alternative option I tried, but found to be less readable at first. However, I have no strong opinion on this. Therefore, I will adapt the pull request according to your wish.

Reply:
I added the node to the end, which made it less readable. With your approach, it looks nice:

```
[gardener-local-ha-multi-zone-worker] Setting up containerd registry mirror for host localhost:5001.
[gardener-local-ha-multi-zone-worker] Setting up containerd registry mirror for host gcr.io.
[gardener-local-ha-multi-zone-worker] Setting up containerd registry mirror for host eu.gcr.io.
[gardener-local-ha-multi-zone-worker] Setting up containerd registry mirror for host ghcr.io.
[gardener-local-ha-multi-zone-worker] Setting up containerd registry mirror for host registry.k8s.io.
[gardener-local-ha-multi-zone-worker] Setting up containerd registry mirror for host quay.io.
[gardener-local-ha-multi-zone-worker] Setting up containerd registry mirror for host europe-docker.pkg.dev.
[gardener-local-ha-multi-zone-control-plane] Setting up containerd registry mirror for host localhost:5001.
[gardener-local-ha-multi-zone-control-plane] Setting up containerd registry mirror for host gcr.io.
[gardener-local-ha-multi-zone-control-plane] Setting up containerd registry mirror for host eu.gcr.io.
[gardener-local-ha-multi-zone-control-plane] Setting up containerd registry mirror for host ghcr.io.
[gardener-local-ha-multi-zone-control-plane] Setting up containerd registry mirror for host registry.k8s.io.
[gardener-local-ha-multi-zone-control-plane] Setting up containerd registry mirror for host quay.io.
[gardener-local-ha-multi-zone-control-plane] Setting up containerd registry mirror for host europe-docker.pkg.dev.
[gardener-local-ha-multi-zone-worker2] Setting up containerd registry mirror for host localhost:5001.
[gardener-local-ha-multi-zone-worker2] Setting up containerd registry mirror for host gcr.io.
[gardener-local-ha-multi-zone-worker2] Setting up containerd registry mirror for host eu.gcr.io.
[gardener-local-ha-multi-zone-worker2] Setting up containerd registry mirror for host ghcr.io.
[gardener-local-ha-multi-zone-worker2] Setting up containerd registry mirror for host registry.k8s.io.
[gardener-local-ha-multi-zone-worker2] Setting up containerd registry mirror for host quay.io.
[gardener-local-ha-multi-zone-worker2] Setting up containerd registry mirror for host europe-docker.pkg.dev.
```

Reply:
Implemented with https://github.com/gardener/gardener/pull/9004/commits/b954976e7670a1bc49d7559e8bd1373f722e3223.

PTAL.